### PR TITLE
Reviewed mapbender QuickStart

### PR DESCRIPTION
**Pull request by flicstar**
Reviewed as part of the QuickStart project during Google Season of Docs. The aim is consistency across all OSGeoLive QuickStarts.
@astroidex Please accept suggestions you agree with. See the template for reference: https://trac.osgeo.org/osgeolive/wiki/How%20to%20document%20the%20quickstart%20file

### CODE REVIEW

**Comment by flicstar** [comment]
This document is too long to be a digestible Quickstart. My suggestions are mostly about removing content (it is valuable but can live elsewhere.) Choose two or three tasks for the user to do to get up and running.

File: doc/quickstart/mapbender_quickstart.rst [hunk 1]
```diff
@@ -433,14 +180,8 @@ What Next?
...
-
-  https://mapbender.org/
-
-You find tutorials at
-
-  https://doc.mapbender.org
+* Mapbender Website https://mapbender.org/
 
-Get involved in the project
+* You find tutorials at https://doc.mapbender.org
```
Comment (astroidex) [inline doc/quickstart/mapbender_quickstart.rst]: Maybe also add: You find an extended tutorial at https://doc.mapbender.org/en/quickstart.html

File: doc/quickstart/mapbender_quickstart.rst [hunk 2]
```diff
@@ -331,96 +168,6 @@ Try it yourself
...
-  .. image:: /images/projects/mapbender/mapbender3_security.png
-     :scale: 70 %
-
-
-Things to try
-================================================================================
-
 Here are some additional challenges for you to try:
 
 #. Try to load some WMS in your application. Try to configure your WMS.
```
Comment (astroidex) [inline doc/quickstart/mapbender_quickstart.rst]: add more challenges like
* Create users and groups and assign an application to the new user/group

File: doc/quickstart/mapbender_quickstart.rst (hunk 1 above)
Comment (flicstar) [reply doc/quickstart/mapbender_quickstart.rst]: done

File: doc/quickstart/mapbender_quickstart.rst (hunk 2 above)
Comment (flicstar) [reply doc/quickstart/mapbender_quickstart.rst]: done.

**Comment by astroidex** [comment]
Looks very good. Ready to be merged.

**Comment by camerons** [comment]
Merged based on review from @astroidex

**Pull request merged**